test/root/scripts/test_attack_element_proxy_fields.txt

```diff
@@ -4,3 +4,9 @@ ash $monster[Knott Yeti].attack_element
 ash $monster[Knott Yeti].attack_elements
 ash $monster[lihc].attack_element
 ash $monster[lihc].attack_elements
+js Monster.get('warwelf').attackElement
```

Comment:
I'd like a test for each changed proxy record if possible. Also if you could instead make a new js file in this directory and add the tests to there that would be better I think.

Just print a bunch of stuff and make a .js.out file for what you expect to be printed. If you can make it human readable in some way, all the better (see test_path.ash for example)

Reply:
I can do that.

Reply:
I created a new test. I could not find a test that actually prints randomModifiers, I think those can happen only during gameplay, right?